docs/data-hub/dataframes.md

```diff
@@ -15,7 +15,7 @@ This page describes how to a Deep Origin DataFrame, which are the primary object
 Create a DataFrame using:
 
 ```python
-from deeporigin import DataFrame
+from deeporigin.data_hub.dataframe import DataFrame
```

Comment:
Does the AI assistant need to update?

Reply:
it's not required. it can still use the old, explicit API:

```
api.get_dataframe
```